### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/discover-our-work#7]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for http://federalist.18f.gov.s3-website-us-east-1.amazonaws.com/site/18f/discover-our-work/
- **Error** A title should be provided for the document, using a non-empty title element in the head section.
  - WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.NoTitleEl
  - `html > head`
  - `<head>
    <link rel="stylesheet" hre...</head>`
### Summary

**Errors:** 1
